study.md

```diff
@@ -115,5 +115,5 @@ alternative. How might you justify this choice of best alternative gem to a
 senior engineer on your team? Explain this below and submit a pull request.
 
 ```md
-<!-- your answer here -->
+At this point, answering these questions is wholly out of my range of knowledge. I was able to find a few gems which have similar functionality to bcrypt, like Strongbox or PBKDF2 (which was made by RSA, an old war horse in cybersec), but all indications point to bcrypt being better. Maybe I bit off a bit more than I can chew in this case.
 ```
```

Comment:
That's a great answer and it's definitely not over your head. We just want you to recognize there's a ton of gems. Some are similar, but we have to weigh the pros and cons of them sometimes!